rust-mozjs/src/glue/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+# Try to get all function signatures to fit on one line, so the jsapi file can be post-processed more easily
+max_width = 500
```

Comment:
I'm wondering if glue.rs should be easy to read - would it be possible to keep the default line length limit and modify the generate_wrappers.sh script instead? Is it only the cases where we have `pub fn method(\n..` that needs to be handled to make this work or is it more complicated?

Reply:
This works even better